app/src/main/java/agency/five/codebase/android/movieapp/ui/home/mapper/HomeScreenMapperImpl.kt

```diff
@@ -0,0 +1,50 @@
+package agency.five.codebase.android.movieapp.ui.home.mapper
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.model.MovieCategory
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelTextViewState
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieCategoryViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieViewState
+import androidx.compose.runtime.mutableStateOf
+
+class HomeScreenMapperImpl() : HomeScreenMapper {
+    override fun toHomeMovieCategoryViewState(
+        movieCategories: List<MovieCategory>,
+        selectedMovieCategory: MovieCategory,
+        movies: List<Movie>,
+    ): HomeMovieCategoryViewState {
+        return HomeMovieCategoryViewState(movieCategories.map { movieCategory ->
```

Comment:
expression function

Reply:
also, this formatting is not correct - parameters in multiple lines should have their names stated